README.md

```diff
@@ -25,7 +25,7 @@ Features:
 - [Installation](#installation)
   - [Docker](#docker)
   - [Conda/Pip venv](#condapip-venv)
-  - [Runpod](#runpod)
+  - [Cloud GPU](#cloud-gpu) - Runpod, Latitude
```

Comment:
Do we need to specify the providers here? If so, does Latitude need some specific instructions like runpod?

Reply:
I was thinking it was a good example for our use case of what those are vs say Azure & AWS